src/presets/eu-core/src/eu-core-print.scss

```diff
@@ -1,2 +1,5 @@
 @use '@ecl/theme-eu-core';
 @use '@ecl/preset-dev/src/dev-print';
+
+// EU specific (temporary solution)
```

Comment:
Checking on `footer-core-ec-print.scss` and `footer-core-eu-print.scss` there is no difference. Why having 2 files when there are no exceptions to handle?

Reply:
Double-checked, still relevant